### PR TITLE
Add simple filtering for quick spot checking

### PR DESCRIPTION
**Pull request by jctanner**

### CODE REVIEW

File: migrate.py [hunk 1]
```diff
@@ -1760,6 +1770,8 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('-s', '--spec', required=True, dest='spec_dir',
                         help='A directory spec with YAML files that describe how to organize collections')
+    parser.add_argument('--filter', dest='filters', action='append',
```
Comment (webknjaz) [inline migrate.py]: Maybe
```suggestion
    parser.add_argument('-e', '--exclude', dest='filters', action='append',
```

Comment (webknjaz) [reply migrate.py]: Oh, wait, it's vice versa. Right?
```suggestion
    parser.add_argument('-o', '--only', dest='filters', action='append',
```

File: migrate.py [hunk 2]
```diff
@@ -965,6 +965,27 @@ def discover_file_migrations(paths, *, find_related=False):
     return copy_map
 
 
+def match_substrings(namespace, collection, plugins, substrings):
+
```
Comment (webknjaz) [inline migrate.py]: Docstring must always go in the first line.
```suggestion
```

**Comment by bcoca** [comment]
@jctanner unsure this will work as you want, also this should be already be possible via a specific scenario.

**Pull request closed without merging**